pkgs/development/python-modules/snakemake/default.nix

```
@@ -0,0 +1,74 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, appdirs
+, configargparse
+, connection-pool
+, datrie
+, filelock
+, GitPython
+, jsonschema
+, nbformat
+, psutil
+, pulp
+, pyyaml
+, ratelimiter
+, smart-open
+, stopit
+, tabulate
+, toposort
+, wrapt
+, pandas
+, pytest
+, requests-mock
+}:
+buildPythonPackage rec {
+  pname = "snakemake";
+  version = "6.15.2";
+
+  src = fetchFromGitHub {
+    owner = "snakemake";
+    repo = pname;
+    rev = "v" + version;
```

most common way to express this:
```suggestion
    rev = "v${version}";
```